mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/tilsagn/model/TilsagnBeregningType.kt

```diff
@@ -4,4 +4,5 @@ enum class TilsagnBeregningType {
     FRI,
     PRIS_PER_MANEDSVERK,
     PRIS_PER_UKESVERK,
+    FAST_SATS_PER_TILTAKSPLASS_PER_MANED,
```

Comment:
Burde de andre nå hete PRIS_PER_TILTAKSPLASS_PER_UKE? 😅 Trenger ikke ta det nå for så vidt